tests/evm/test_extcodecopy.py

```diff
@@ -0,0 +1,167 @@
+import pytest
+from itertools import chain
+from zkevm_specs.copy_circuit import verify_copy_table
+from zkevm_specs.evm import (
+    verify_steps,
+    Tables,
+    Block,
+    Bytecode,
+    ExecutionState,
+    StepState,
+    Opcode,
+    U64,
+    U160,
+    RLC,
+    CopyCircuit,
+    CopyDataTypeTag,
+    AccountFieldTag,
+    CallContextFieldTag,
+)
+from zkevm_specs.evm.typing import RWDictionary
+from zkevm_specs.util import (
+    rand_fq,
+    keccak256,
+    EXTRA_GAS_COST_ACCOUNT_COLD_ACCESS,
+    GAS_COST_COPY,
+    memory_word_size,
+    memory_expansion,
+)
+
+TESTING_DATA = (
```

Comment:
Could you also add test cases where the code length is more than 256? This is to test that when codesize does not fit into a single byte, your gadget should still work OK, indicating that there is no mismatch in FQ/RLC values being used.

Reply:
And also tests with `length` field (`length` for extcodecopy) where `length > 256`. I don't see a place where these tests should fail. But better to have them in place so subsequent edits made to this also satisfy those test cases!

Reply:
I added some test cases in [cc950ff](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/257/commits/cc950ffa70beac3bb602439daf340f03bd243dbd) for `code length > 256` and out of bound cases. Would you please check? :)

Reply:
@DoHoonKim8 I meant, a test case where the code size is also > 256. So you will need `rand_bytes(n)` where `n > 256` (instead of 64).

Reply:
I'm bit confused. 

> Could you also add test cases where the code length is more than 256? This is to test that when codesize does not fit into a single byte

I thought that we need a test case when the code length is more that 256 "bits". I used `rand_bytes(64)` to generate 64 bytes length code, which is bigger than 256 bits. Umm... sorry for my misunderstanding, do I misunderstanding by any chance?